packages/nitro-protocol/src/signatures.ts

```diff
@@ -71,7 +71,7 @@ function hashMessage(hashedData: string): string {
   return utils.hashMessage(utils.arrayify(hashedData));
 }
 
-function signData(hashedData: string, privateKey: string): Signature {
+export function signData(hashedData: string, privateKey: string): Signature {
   const signingKey = new utils.SigningKey(privateKey);
 
   return utils.splitSignature(signingKey.signDigest(hashMessage(hashedData)));
```

Comment:
Why is the argument called `hashedData` when we pass it into `hashMessage`? I forget if we're intentionally double hashing, or if the naming is just confusing

Reply:
We are double hashing.